package.json

```diff
@@ -101,8 +101,10 @@
     "@vanilla-extract/babel-plugin": "^1.1.7",
     "@vanilla-extract/webpack-plugin": "^2.1.11",
     "cypress": "10.3.1",
+    "cypress-hardhat": "^1.0.0",
     "env-cmd": "^10.1.0",
     "eslint": "^7.11.0",
+    "hardhat": "^2.14.0",
```

Comment:
`hardhat` is a _large_ dependency, so it is expected that there is a lot of churn in `yarn.lock`.